lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -312,9 +313,22 @@ nixShellRunConfig root isPure command = do
       , [cs]
       ])
 
+-- | Escape using ANSI C-style quotes $''
```

Comment:
Does this Haddock compile? I'd assume you need `@$''@`

Reply:
:fish: :fist_left: i'll check